tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
```

Comment:
Maybe target something earlier, like `es5`?

Reply:
I can push it back to es2017. We used `async` and `await` in the javascript version and nobody complained about it not working last year. caniuse says it's been supported by everything since 2017.